airbyte-integrations/connectors/source-bigquery/src/test/resources/test_config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dataset_id": "dataset",
+  "project_id": "project",
+  "credentials_json": "credentials"
+}
```

Comment:
EOF, can you apply `./gradlew format` to ensure everything is formatted corretly?

Reply:
Ok, done.